pykern/pkcli/fmt.py

```diff
@@ -17,13 +18,19 @@ def run(path):
     _black(path)
 
 
-def diff(path):
+def diff(path_or_paths):
```

Comment:
`def diff(*paths)` (see below). This allows calling with multiple paths from the command line.

Reply:
81558f6e653269c07a7286f77898de84b03955fc